src/component/Tooltip.tsx

```diff
@@ -90,82 +90,78 @@ export type TooltipProps<TValue extends ValueType, TName extends NameType> = Def
   useTranslate3d?: boolean;
 };
 
-export class Tooltip<TValue extends ValueType, TName extends NameType> extends PureComponent<
-  TooltipProps<TValue, TName>
-> {
-  static displayName = 'Tooltip';
-
-  static defaultProps = {
-    active: false,
-    allowEscapeViewBox: { x: false, y: false },
-    reverseDirection: { x: false, y: false },
-    offset: 10,
-    viewBox: { x1: 0, x2: 0, y1: 0, y2: 0 },
-    coordinate: { x: 0, y: 0 },
-    cursorStyle: {},
-    separator: ' : ',
-    wrapperStyle: {},
-    contentStyle: {},
-    itemStyle: {},
-    labelStyle: {},
-    cursor: true,
-    trigger: 'hover',
```

Comment:
Why remove these two?

Reply:
which two?
`cursor: true,` is still there

`trigger: 'hover',` - I forget, might just be a miss (though it does get set without this defaulted)

will add it back